package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "private": true,
+  "license": "UNLICENSED",
+  "devDependencies": {
+    "@babel/core": "^7.3.4",
+    "@babel/preset-env": "^7.3.4",
+    "@babel/preset-react": "^7.0.0",
+    "acorn": "^6.1.1",
+    "acorn-jsx": "^5.0.1",
+    "babel-preset-react": "^6.24.1",
+    "commander": "^2.19.0",
+    "contentful-cli": "^0.24.0",
+    "eslint": "^5.15.1",
+    "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-jest": "^22.3.0",
+    "eslint-plugin-node": "^8.0.1",
+    "eslint-plugin-promise": "^4.0.1",
+    "eslint-plugin-react": "^7.12.4",
+    "espree": "^5.0.1",
+    "jest": "^24.3.1",
+    "log-symbols": "^2.2.0",
+    "make-dir": "^2.1.0",
+    "netlify": "^2.4.1",
+    "ora": "^3.2.0",
+    "parcel-bundler": "^1.12.2",
+    "react-test-renderer": "^16.8.4"
+  },
+  "scripts": {
+    "build": "NODE_ENV=production node ./scripts/build.js",
+    "start": "contentful space use && NODE_ENV=development node ./scripts/start.js",
+    "deploy": "node ./scripts/deploy.js",
+    "lint": "eslint marketplace/**/*.js scripts/**/*.js",
+    "test": "jest marketplace/**/test/*.test.js",
+    "coverage": "jest --coverage marketplace/**/test/*.test.js"
+  },
+  "engines": {
+    "node": ">=10.15.0"
+  },
+  "dependencies": {
+    "@contentful/forma-36-fcss": "0.0.13",
+    "@contentful/forma-36-react-components": "^2.9.8",
+    "@contentful/rich-text-plain-text-renderer": "^13.1.0",
+    "alex": "^7.1.0",
```

Comment:
did I miss something or we defined only one package.json file for the whole repo?

Reply:
Yeah I build it with a single package.json. It makes it easier to share things, especially for testing and linting. Definitely possible to break this up into individual packages, just a bit more involved in terms of set up.

I guess it comes down to how we want to maintain these extensions. If there's a new version of say Forma 36 released, do we wanna update it all in one go or do we see each extension as a fully separate project maintained independently?

Reply:
I think each extension should be self contained, so we can change dependencies per extension.